modules/core/src/core/install/commands/install.ts

```diff
@@ -100,7 +100,7 @@ export const handler: Handler<Args> = async function handler(argv, { graph, logg
 	await sudo({
 		name: 'Create executable',
 		cmd: 'echo',
-		args: [`"#!/bin/zsh\n\n${process.argv[1]} \\$@"`, '|', 'sudo', 'tee', installLocation],
+		args: [`"#!/bin/zsh\n\n${process.argv[1].replace(' ', '\\ ')} \\$@"`, '|', 'sudo', 'tee', installLocation],
```

Comment:
doesn't this replace only the first space? would need to add `/g` i believe to make it replace all open spaces